relay/components/Navigation/NavBar.tsx

```diff
@@ -77,9 +42,35 @@ export const Nav = () => {
                   </a>
                 </nav>
               </div>
-              <div className="px-12 lg:px-0">
-
-                <div className="max-w-xs mx-auto w-full lg:max-w-md">
+              {/* @TODO figure out a way to right align login button without dynamic js below, and without breaking
```

Comment:
I think `float-right` might work here.

Reply:
ahh indeed, unfortunately already tried that one, `right-0` and a few others, it seems that the parent flex & grid elements are overriding that.

This is what it ends up looking like:

![image](https://user-images.githubusercontent.com/2301666/125830902-b8d2a963-15d7-473e-a541-559f1aa535bd.png)